app/src/main/java/com/amaze/filemanager/adapters/AppsAdapter.java

```diff
@@ -253,10 +254,9 @@ private void showPopup(View v, final AppDataParcelable rowItem){
     private final boolean isGooglePlayStoreAvailable() {
         PackageManager pm = context.getPackageManager();
         try {
-            pm.getPackageInfo(COM_ANDROID_VENDING, PackageManager.GET_ACTIVITIES);
-            return true;
+            PackageInfo appInfo = pm.getPackageInfo(COM_ANDROID_VENDING, PackageManager.GET_ACTIVITIES);
+            return appInfo != null || appInfo.activities != null;
```

Comment:
Or? If `appInfo` is null `appInfo.activities` will NPE.

Reply:
Oops.